taca/__init__.py

```diff
@@ -1,4 +1,4 @@
 """ Main TACA module
 """
 
-__version__ = '0.5.0'
+__version__ = '0.5.1'
```

Comment:
There is a conflict due to this, current [version](https://github.com/SciLifeLab/TACA/blob/master/taca/__init__.py) number is not `0.5.0`